src/org/jitsi/impl/neomedia/rtp/translator/RTPTranslatorImpl.java

```diff
@@ -225,6 +227,12 @@ public static int readUnsignedShort(byte[] buf, int off)
     private long localSSRC = -1;
 
     /**
+     *
+     */
+    private final RecurringProcessibleExecutor recurringProcessibleExecutor
```

Comment:
@gpolitis, please add javadocs on the RecurringProcessibleExecutor field and getter. I know what RecurringProcessibleExecutor does in general but I'd like it to be obvious why RTPTranslatorImpl is a provider of RecurringProcessibleExecutor.